packages/editor/src/components/post-type-support-check/index.js

```diff
@@ -27,7 +27,7 @@ function PostTypeSupportCheck( { children, supportKeys } ) {
 		const { getPostType } = select( coreStore );
 		return getPostType( getEditedPostAttribute( 'type' ) );
 	}, [] );
-	let isSupported = true;
+	let isSupported = !! postType;
```

Comment:
This is a small logic check where post type supports return "false" if the postType object is not defined/loaded yet. I think it's a better default as otherwise we'll see some panels appear when the page is being loaded and they disappear once the post type loads.

Reply:
I thought the original logic was intentional. I remember investigating #29407 and coming across the original PR #7748.

> This requires that we change the default behavior of PostTypeSupportCheck to assume true if the post type is not yet known. I expect that this is probably the majority case anyways.

Reply:
I think the original behavior was misguided personally. I prefer something to not show up until we're certain that it's useful over something that disappears under me (which happens on the site editor if you reload the page).

WDYT?

Reply:
And yes as suggested in #29407 there's an actual risk of JS errors and breaking the editor if you render something (even for a millisecond) in a post type that have a property that is undefined or something.

Reply:
I'm also in favor of the new logic. Let's close #29407 when this is merged.

P.S. I'm sorry I didn't have time to review this properly today, but it's on my list for Monday.